crates/optimism/node/src/args_xlayer.rs

```diff
@@ -20,8 +26,9 @@ pub struct XLayerArgs {
 impl XLayerArgs {
     /// Validate all X Layer configurations
     pub fn validate(&self) -> Result<(), String> {
-        self.intercept.validate()
+        self.intercept.validate()?;
```

Comment:
why need to change this?

Reply:
add this function:  self.gas_price.validate()